common/web/gesture-recognizer/src/inputEventEngine.ts

```diff
@@ -1,59 +1,52 @@
 /// <reference path="inputEventCoordinate.ts" />
+/// <reference path="gestureRecognizerConfiguration.ts" />
+/// <reference path="includes/events.ts" />
 
 namespace com.keyman.osk {
+  type Mutable<Type> = {
+    -readonly [Property in keyof Type]: Type[Property];
+  };
```

Comment:
You've got this here and in gestureRecognizer.ts. It should be in its own mutable.ts I think? Where did this come from? Is it a common pattern?

Reply:
Seems common enough that TypeScript bothered to support it.

I don't think this was the original post that led me to it, but... https://stackoverflow.com/a/62038378.